src/main/java/com/kryptnostic/rhizome/core/JettyAnnotationConfigurationWorkaround.java

```diff
@@ -0,0 +1,44 @@
+package com.kryptnostic.rhizome.core;
```

Comment:
is this movable to a different package?

Reply:
I've wanted to move this package into com.geekbeast.rhizome.core for a while now. Should be a separate PR because it will trigger a lot of downstream renames.

Reply:
This one is safe to move, because it's not referenced externally so I went ahead and did it.